motion_predict/README.md

```diff
@@ -1,5 +1,16 @@
 # motion_predict
 
-This library implements some basic motion prediction functions meant to support external object handling under the motion_predict namespace. Currently there are two motion models implements a Constant Velocity (CV) model and a Constant Turn-Rate and Velocity (CTRV) model. See the motion_predict.h and predict_ctrv.h files for the relevant interfaces. 
+This library implements some basic motion prediction functions meant to support external object handling under the motion_predict namespace. Currently there are two motion models implements a Constant Velocity (CV) model and a Constant Turn-Rate and Velocity (CTRV) model. See the motion_predict.h and predict_ctrv.h files for the relevant interfaces.
 
-Link to detailed design on Confluence: https://usdot-carma.atlassian.net/l/c/e3U5DXZi
\ No newline at end of file
+> [!NOTE]
+> Both motion models expect the objects' twist.linear data to be in a map frame as opposed to base_link frame commonly used in ROS.
```

Comment:
I think it is common to deal with external objects in the `map` frame, whereas ego vehicle motion is typically handled in the `base_link` frame. Since `base_link` is based off of a rigid transform from the ego CARMA Platform vehicle, even if `base_link` frame was used, we would still require an external object's `twist.linear.x` and `twist.linear.y` for computing its heading.

I don't have a change to request here, but wanted to type my thoughts out on it.

Reply:
I see the confusion I made here. The base_link frame I mentioned here is the orientation frame of the object